src/dehtml.rs

```diff
@@ -457,7 +457,7 @@ mod tests {
 
     #[test]
     fn test_dehtml_parse_href() {
-        let html = "<a href=url>text</a";
```

Comment:
This test doesn't work anymore. It was parsed as "[text". Do we want to support such truncated HTML?

Reply:
I also think so. It was added in d7d7147549